services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events.py

```diff
@@ -463,10 +404,42 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
         dynamic_sidecar_client = get_dynamic_sidecar_client(app)
         dynamic_sidecar_endpoint = scheduler_data.endpoint
 
+        # check values have been set by previous step
```

Comment:
Yes, I undestand the reason behind your question. Now that I read it I also think the same.
If I remember correctly this might be one of the first pieces of code added. I will leave it here, in the near future this will go away.